src/sentry/api/serializers/models/dashboard.py

```diff
@@ -143,11 +143,26 @@ def get_attrs(self, item_list, user):
         return result
 
     def serialize(self, obj, attrs, user, **kwargs):
+        page_filter_keys = ["environment", "range", "start", "end"]
+        dashboard_filter_keys = ["releases"]
         data = {
             "id": str(obj.id),
             "title": obj.title,
             "dateCreated": obj.date_added,
             "createdBy": serialize(obj.created_by, serializer=UserSerializer()),
             "widgets": attrs["widgets"],
+            "projects": [project.id for project in obj.projects.all()],
         }
+
+        if obj.filters is not None:
+            for key in page_filter_keys:
+                if obj.filters.get(key) is not None:
+                    data[key] = obj.filters[key]
+
+            for key in dashboard_filter_keys:
+                if obj.filters.get(key) is not None:
+                    if data.get("filters") is None:
```

Comment:
@narsaynorath you can move this out of the for loop, and initialize it beforehand.

Reply:
At first we weren't sure if we wanted an empty dict in the response if there weren't filter values, but we went with an empty dict as you suggested because it's more explicit than hiding the key